news.html

```diff
@@ -1,3 +1,12 @@
+<span>
+    <h4> 
+        <a target="_blank" rel="noopener noreferrer"
+           href="https://medium.com/dockstore/introduction-to-dockstore-a55ae101d707">
+        Introducing the Dockstore Blog</a>
+    </h4>
+    <span>We have published our first blog post and we are very excited to connect with current and potential Dockstore users through this medium. Read and follow the blog to learn more about Dockstore.</span>
```

Comment:
> ...to connect with current and potential Dockstore users... 

I suggest dropping `potential`, since they are already on the web site. Then if you do that, there's no need for `current`, so then you would have:

`...to connect with Dockstore users...`

Reply:
Done!